sdk/core/amqp-common/src/retry.ts

```diff
@@ -97,12 +97,23 @@ async function checkNetworkConnection(host: string): Promise<boolean> {
   if (isNode) {
     return new Promise((res) => {
       resolve(host, function(err: any): void {
-        // List of possible DNS error codes: https://nodejs.org/dist/latest-v12.x/docs/api/dns.html#dns_error_codes
-        if (err && (err.code === "ECONNREFUSED" || err.code === "ETIMEOUT")) {
-          res(false);
+        if (err) {
```

Comment:
How about a log just before `resolve` is called?
Should help in indicating that the resolve call is in progress
Especially helpful when resolve takes longer to complete

Reply:
Done!